RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2181,32 +2197,52 @@ boolean L11_hiddenCityZones()
 		if (canUseMachete && !equipMachete()) {
 			return false;
 		}
+		if(!canUseMachete && !equipTearawayPants())
```

Comment:
if we don't have tearaway pants, or if we're in pokefam, the equip function will always return false and so we'll never adventure at the shrines? Or am I misinterpreting how the code works?